src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -16,6 +16,7 @@ import {
 import {ChatStatus} from '../../../types/database/schema/ChannelList.types';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {calculateTime} from '../../utils/time';
+import {handleUserName} from '../../utils/string/StringUtils';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new import statement for `handleUserName` function from `StringUtils` has been added. This is a good practice if the function is not tied to any specific component and can be used across multiple components.
```diff
+ import {handleUserName} from '../../utils/string/StringUtils';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->